docs/topics/api/auth_internal.rst

```diff
@@ -39,6 +39,35 @@ The token is available in two forms:
     * For all endpoints, as a cookie called ``api_auth_token``. This cookie
       expires after 30 days and is set as ``HttpOnly``.
 
+The response will contain some profile data for personalization:
+
+    :>json int id: The numeric user id.
+    :>json string email: Email address used by the user to log in and create this account.
+    :>json string name: The name chosen by the user, or the username if not set.
+    :>json string picture_url: URL to a photo of the user, or `/static/img/anon_user.png` if not set.
+    :>json string username: username chosen by the user, used in the account url. If not set will be a randomly generated string.
```

Comment:
Not really... it's set on the model save first time they login/register.  

There is a `has_anonymous_username` method on User and it's just `return re.match('^anonymous-[0-9a-f]{32}$', self.username)`.  You could implement that in frontend, or we could expose that method in the API (different issue preferably as it should be in the standard account API too)